wallet-webapps-common/src/main/webapp/vue-app/components/SendTokensForm.vue

```diff
@@ -629,11 +638,12 @@ export default {
     close(){
       this.$refs.sendTokensForm.close();
     },
-    showAlert(alertType, alertMessage, alertTransactionHash){
+    showAlert(alertType, alertMessage, alertTransactionHash, alertShowTransactionLink){
```

Comment:
no need to introduce a new variable here. In fact, when the transaction hash is empty, it means, no need to display Transaction link, else display it.

Reply:
would you please check the changes I made out ! @boubaker